backend/src/lib/api-docs/constants.ts

```diff
@@ -1,3 +1,12 @@
+export const DEFAULT_REQUEST_SCHEMA = {
```

Comment:
Instead of this, i think it is better to follow this https://fastify.dev/docs/latest/Reference/Validation-and-Serialization/#adding-a-shared-schema